test/animations.html

```diff
@@ -29,8 +29,7 @@
 
       beforeEach(() => {
         overlay = fixture('default');
-
-        sinon.stub(overlay, '_shouldAnimate', () => true);
+        overlay.style.setProperty('animation-name', 'close-anim');
```

Comment:
note: is this workaround needed by the new test only, or the other tests need it too?

Reply:
Only required by the new test. Should be true by default with vaadin overlay (assuming the lumo version is used in tests)

Reply:
Ok makes sense, thanks.